lib/kafo/configuration.rb

```diff
@@ -23,14 +25,20 @@ class Configuration
         :answer_file          => './config/answers.yaml',
         :installer_dir        => '.',
         :module_dirs          => ['./modules'],
-        :colors               => Kafo::ColorScheme.colors_possible?,
```

Comment:
Don't we still need a default for this? If I read this code correctly, I think we'll end up without colors if you drop this:
https://github.com/theforeman/kafo/blob/9fb119aff2f9827ecd088438aacc613f3c56c8c1/lib/kafo/kafo_configure.rb#L498-L506

For my own understanding, I wrote https://github.com/theforeman/kafo/pull/296. Currently untested, but I think that would be needed.